teamA/jeongDoYoung/app.js

```diff
@@ -0,0 +1,43 @@
+const http = require("http");
+const morgan = require("morgan");
+const express = require("express");
+const cors = require("cors");
+const dotenv = require("dotenv");
+
+dotenv.config();        //환경변수에 적어뒀던 수많은 typeorm과 관련된 수많은 내용들을 app.js에서 사용할 수 있게 만듬
```

Comment:
불필요한 주석은 모두 제거 해주세요 !
```suggestion
dotenv.config();
```